### ISSUE TITLE
option to replace firefox (rpm) with firefox from flathub

### ISSUE THREAD

**Issue by iaacornus** [iaacornus/Fedora-OSTree-Setup#33]

**Comment by trytomakeyouprivate** [comment]
for me firefox had weird tab crashes so I switched back to the Flatpak one, no problems.

**Comment by iaacornus** [comment]
> for me firefox had weird tab crashes so I switched back to the Flatpak one, no problems.

yeah this would allow user to replace firefox rpm with firefox flatpak

**Comment by iaacornus** [comment]
with utter seriousness, why not we just build a silverblue image with all of these modification @trytomakeyouprivate

**Comment by trytomakeyouprivate** [comment]
> with utter seriousness, why not we just build a silverblue image with all of these modification @trytomakeyouprivate

hahaha I dont think we are thaat far. Its just a few settings and a few overlays right?

I dont know that much about python, so I would be very happy if you could make the start, as I was a bit confused by some stuff. If I can read it I am sure I can just build in all the elements and that script should be ready soon!

I mean, we would need a server, a program auto-applying the changes, intense bandwidth, a website hell no XD

And *cough* never gonna use Gnome 🏃

**Comment by iaacornus** [comment]
> > with utter seriousness, why not we just build a silverblue image with all of these modification @trytomakeyouprivate
> 
> hahaha I dont think we are thaat far. Its just a few settings and a few overlays right?
> 
> I dont know that much about python, so I would be very happy if you could make the start, as I was a bit confused by some stuff. If I can read it I am sure I can just build in all the elements and that script should be ready soon!
> 
> I mean, we would need a server, a program auto-applying the changes, intense bandwidth, a website hell no XD
> 
> And _cough_ never gonna use Gnome runner

understandable, it seems like i've underestimated the problem.

**Comment by iaacornus** [comment]
also should remove `firefox-langpacks` along with `firefox` package
